.github/chainguard/release.sts.yaml

```diff
@@ -1,7 +1,7 @@
 issuer: https://token.actions.githubusercontent.com
-subject: repo:chainguard-dev/bincapz:ref:refs/heads/main
+subject: repo:chainguard-dev/malcontent:ref:refs/heads/main
```

Comment:
This shouldn't be an issue since we'll rename the repository right after the PR merges.